src/core.ts

```diff
@@ -65,36 +66,41 @@ export function activate() {
     onActivateError(e);
     return;
   }
+
+  state = State.Activated;
```

Comment:
What motivated this change? What bug will it fix?

Reply:
Before this fix attempting to teardown clarity instance, which was never activated caused a JS error.

---
In reply to: [158272415](https://github.com/Microsoft/clarity-js/pull/45#discussion_r158272415) [](ancestors = 158272415)